recipes/typing-extensions/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "typing-extensions" %}
+{% set version = "3.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1c0a8e3b4ce55207a03dd0dcb98bc47a704c71f14fe4311ec860cc8af8f4bd27
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - typing >=3.6.2  # [py<3.5]
```

Comment:
Pity about this selector making it not noarch

Reply:
The source [is actually different](https://github.com/python/typing/blob/ff2084eb9ab5eea778d744c706e6a6aad05ea4bd/typing_extensions/setup.py#L47) for python 2 and 3 anyway.